SmartDeviceLink/SDLSoftButtonManager.m

```diff
@@ -65,8 +66,7 @@ - (instancetype)initWithConnectionManager:(id<SDLConnectionManagerType>)connecti
     _transactionQueue = [self sdl_newTransactionQueue];
     _batchQueue = [NSMutableArray array];
 
-    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(sdl_registerResponse:) name:SDLDidReceiveRegisterAppInterfaceResponse object:nil];
-    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(sdl_displayLayoutResponse:) name:SDLDidReceiveSetDisplayLayoutResponse object:nil];
+    [_systemCapabilityManager subscribeToCapabilityType:SDLSystemCapabilityTypeDisplays withObserver:self selector:@selector(sdl_displayCapabilityUpdate:)];
```

Comment:
This always fails because the `SystemCapabilityManager.isStreamingSupported` check in `SystemCapabilityManager.subscribeToCapabilityType` fails. Two solutions I can think of:

1. This check always succeeds if checking `DISPLAYS`
2. This check is removed entirely

I think I lean toward (1) right now. What do you think?

Reply:
It is implemented as:
```
    [_systemCapabilityManager subscribeToCapabilityType:SDLSystemCapabilityTypeDisplays withObserver:self selector:@selector(sdl_displayCapabilityUpdate:)];
    [[NSNotificationCenter defaultCenter] addObserver:self selector:@selector(sdl_hmiStatusNotification:) name:SDLDidChangeHMIStatusNotification object:nil];

```